AC-1394 Review Glue API backend docs (#1269)
spryker/spryker-docs: docs/scos/dev/glue-api-guides/202204.0/glue-backend-api/how-to-guides/how-to-use-authorization-framework.md

---
title: How to use authorization framework

> description: This document shows how to create a new authorization strategy.

Reviewer: ```suggestion
description: This document describes how to create a new authorization strategy.
```

last_updated: September 30, 2022
template: howto-guide-template
redirect_from:
  - /docs/scos/dev/glue-api-guides/202204.0/glue-backend-api/how-to-guides/using-authorization-framework.html
---

> This document shows how to create a new authorization strategy.

Reviewer: ```suggestion
This document describes how to create a new authorization strategy.
```

> Integrate authorization following this guide [https://docs.spryker.com/docs/scos/dev/technical-enhancement-integration-guides/integrating-authorization-enabler.html](https://docs.spryker.com/docs/scos/dev/technical-enhancement-integration-guides/integrating-authorization-enabler.html).

Author: ```suggestion
Integrate authorization following the [Integrating Authorization Enabler](/docs/scos/dev/technical-enhancement-integration-guides/integrating-authorization-enabler.html) guide.
```

> 

Reviewer: Please remove the link from the text and add a full name of the article

> The first step is creating a strategy that is a plugin responsible for performing the authorization itself:

Reviewer: ```suggestion
The first step is creating a strategy that is a plugin responsible for performing the authorization:
```

        return static::STRATEGY_NAME;
    }
}
```
</details>

This plugin is run by `AuthorizationClient::authorize()`.

> There is a way to connect the resources and custom routes with the just-created strategy. They need to implement `AuthorizationStrategyAwareResourceRoutePluginInterface` pointing to the strategy:

Reviewer: ```suggestion
There is a way to connect the resources and custom routes with this strategy. They need to implement `AuthorizationStrategyAwareResourceRoutePluginInterface` pointing to the strategy:
```

        ];
    }
}
```

> | FIELD IN ROUTE AUTHORIZATION CONFIG TRANSFER | MEANING |

Reviewer: ```suggestion
| FIELD IN ROUTE AUTHORIZATION CONFIG TRANSFER | DESCRIPTION |
```
